Pipfile.lock

```diff
@@ -28,19 +28,18 @@
         },
         "bx-py-utils": {
             "hashes": [
-                "sha256:508cfc1d0fa6c22298f697c4efaa913337847d488d8a53eeccfae9ee106123f6",
-                "sha256:c92ebc4fb122e3e3c228d984d0a1f5c3284c3da6aab1a1c753f7eb1f71bdab3a"
+                "sha256:6d9211f0d01a51bb83cc260b4a5293b8364ce539d60a59975fc6d95e103e9041",
+                "sha256:d603449d5e481035ae1186b34e57465f9e5f8235c4f7f1a73b98899a475b2b28"
             ],
-            "markers": "python_version >= '3.10' and python_version < '4'",
-            "version": "==104"
+            "version": "==93"
```

Comment:
This is a downgrade, because bx_py_utils supports only Python >= 3.10... I think we should also change bx_django_utils and support also only >= 3.10, isn't it?!?